docs/platforms/javascript/common/migration/v7-to-v8/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Migrate from 7.x to 8.x
+sidebar_order: 8999
+description: "Learn about migrating from Sentry JavaScript SDK 7.x to 8.x"
+---
+
+The main goal of version 8 is to improve our performance monitoring APIs, integrations API, and ESM support. This
+version is breaking because we removed deprecated APIs, restructured npm package contents, and introduced new
+dependencies on OpenTelemetry.
+
+## Migration Codemod
+
+Before updating to `8.x` of the SDK, we recommend upgrading to the latest version of `7.x`. To fix all deprecations on `7.x`, you can use the `@sentry/migr8` codemod to automatically update your SDK usage. `@sentry/migr8` requires Node 18+.
+
+```bash
+npx @sentry/migr8@latest
+```
+
+This will let you select which updates to run, and automatically update your code. Make sure to still review all code changes!
+
+## Migration Guide
+
+Please see our [detailed migration guide](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#upgrading-from-7x-to-8x) for more information on migrating from `7.x` to `8.x`.
+
+{/* Make sure to update the PlatformSection below when new fully fleshed out migration guides are added */}
+
+<PlatformSection supported={["javascript.nextjs"]}>
+
+## Upgrading to `8.x`
+
+<PlatformContent includePath="migration/javascript-v8/intro" />
+
+We recommend you read through all the [Important Changes](#important-changes) as they affect all Next.js SDK users. The [Other Changes](#other-changes) linked below only affect users who have very custom instrumentation. There is also a [Troubleshooting](#troubleshooting) section for common issues.
```

Comment:
Not sure if we want/should do this in this PR or in a follow up, but we should make sure to only show the relevant stuff for the current guide (e.g. replay can be hidden for node guides, profiling-node for non-node guides etc).

Reply:
Yes! I moved replay into browser specific other changes